### ISSUE TITLE
Segmentation fault with caps lock active (default example bar)

### ISSUE THREAD

**Issue by hjpotter92** [jaagr/polybar#1665]
## Describe the issue
The example bar initialises normally when I launch it. However, there is a segmentation fault if either:

 - CAPS lock is ON before launching
 - CAPS lock is turned ON after launching polybar

After disabling the num lock black list, the same behaviour is noticed for that key status as well.

**Expected behavior:**

It should not crash.

**Was it working before?**
Did not try an older version. Installed using the AUR.

## To Reproduce
Use the example bar config. Set numlock/capslock as active, and try launching the bar.

## Polybar Log

```
➜ polybar example
warn: No monitor specified, using "DP-4"
error: Disabling module "bspwm" (reason: Could not find socket: /tmp/bspwm_0_0-socket)
error: module/xbacklight: Could not get data (err: XCB_NAME (15))
error: Disabling module "xbacklight" (reason: Not supported for "DP-4")
warn: pulseaudio: using default sink alsa_output.pci-0000_01_00.1.hdmi-stereo-extra2
error: Disabling module "wlan" (reason: Invalid network interface "net1")
error: Disabling module "battery" (reason: No suitable way to get current charge state)
error: Disabling module "temperature" (reason: The file '/sys/class/thermal/thermal_zone0/temp' does not exist)
[1]    5360 segmentation fault (core dumped)  polybar example
```

## Environment:
* WM: i3 gaps
* Output of `polybar -vvv`:

```
➜ polybar -vvv
polybar 3.3.0-35-gdb9a83a8

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) -render -damage -sync +composite +xkb +xrm +xcursor

Build type: RelWithDebInfo
Compiler: /bin/c++
Compiler flags: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wall -Wextra -Werror -Wno-noexcept-type -pedantic -pedantic-errors
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
```

**Comment by patrick96** [comment]
Duplicate of #1660

**Comment by patrick96** [comment]
Fixed in #1661
